Set a reading order to the compatibility import file

agamrafaeli opened the pull request:
The order is as follows:
1. Imports, in the same format of "import X"
2. Members denoting versions of python
3. Members of imports that are shadowed for usage across the system
4. Global variables
5. If conditions of imports. (internally organized by this order)
6. Function definitions

agamrafaeli committed:
Set a reading order to the compatability import file

The order is as follows:
1. Imports, in the same format of "import X"
2. Members denoting versions of python
3. Members of imports that are shadowed for usage across the system
4. Global variables
5. If conditions of imports. (internally organized by this order)
6. Function definitions

agamrafaeli commented:
Happy to hear other conventions and suggestions for order and then do the grunt work on my own.

sqla-tester reviewed (commented):
OK, this is **sqla-tester** setting up my work to try to get this PR into gerrit so we can run tests and reviews and stuff

sqla-tester commented:
Change has been squashed to Gerrit review: https://gerrit.sqlalchemy.org/#/c/sqlalchemy/sqlalchemy/+/1036

zzzeek commented on lib/sqlalchemy/util/compat.py:
```diff
@@ -327,12 +302,32 @@ def nested(*managers):
                 exc = sys.exc_info()
         if exc != (None, None, None):
             reraise(exc[0], exc[1], exc[2])
+    exc = (None, None, None)
```
this line moving from up above is what broke the tests in https://jenkins.sqlalchemy.org/job/sqlalchemy_gerrit/cext=cext,db=sqlite-postgresql,pyv=py27/1930/#showFailuresLink, can you move that back?   you can re-push to this PR and I should be able to continue from there.

agamrafaeli committed:
Correct location of line

In the last commit a line was mistakenly misplaced. This commit
correctly places that line.

sqla-tester reviewed (commented):
OK, this is **sqla-tester** setting up my work to try to get this PR into gerrit so we can run tests and reviews and stuff

sqla-tester commented:
Change has been squashed to Gerrit review: https://gerrit.sqlalchemy.org/#/c/sqlalchemy/sqlalchemy/+/1036

sqla-tester commented:
Gerrit review https://gerrit.sqlalchemy.org/1036 has been **merged**. Congratulations! :)
